### ISSUE TITLE
Bringing the "Save Detail" modal back (or, adding a nicer GUI for batch.html)

### ISSUE THREAD

**Issue by onpaws** [danny0838/webscrapbook#239]
Hello,
Long time Scrapbook X user here, thank you for your efforts making such a nice tool!

Like others here we have been happy users of legacy ScrapBook for some years, keeping the old Firefox v59 around to use it. Recently we consider migrating to WebScrapBook.

<summary>
A particular feature of the old ScrapBook we appreciate is the "Save Detail" modal (screenshot included).
<details>

![image](https://user-images.githubusercontent.com/182515/136291143-19a04584-b6df-4fd6-adaf-a828f9ab1c3f.png)

</details>
</summary>
One reason we like this modal is because it's nice being able to set these details _before_ starting the (possibly long) page capture.

<br /><br />
<summary>
When checking out WebScrapBook, we noticed the "Capture tabs as..." menu option...
<details>

![image](https://user-images.githubusercontent.com/182515/136291790-3ff94e0e-8607-41a1-98bc-f31b38764089.png)

</details>
</summary>

...brings a sort of "JSON version" of this feature. 

<summary>

<details>

![image](https://user-images.githubusercontent.com/182515/136291261-880eaf1a-ab40-4185-b5e3-b3cddba48932.png)

</details>
</summary>

If I understand correctly, [batch.html](https://github.com/danny0838/webscrapbook/blob/ca337e0371688c313adb468ddaf0cc1ef7f1f157/src/capturer/batch.html#L1) is the relevant file, and it currently renders [DEFAULT_OPTIONS](https://github.com/danny0838/webscrapbook/blob/3c4a5e0b23e782574b418c09ed115f841697bbbb/src/core/common.js#L47) into a `<textarea>`. This seems to let users customize the options before the batch job starts. 
This is a great start! 💯  

The issue we're facing is with our less technical users. They get intimidated here and I was wondering if there is an appetite to bring something closer to the old "Save Details" UI into batch.html.

Basically, we'd like to let users' ability to choose the `parentId` to be driven by a GUI similar to the old ScrapBook days.
What do you think?
Thanks again for your efforts 👍

**Comment by danny0838** [comment]
No. We don't have a hard limit here. But we may reject or ask for correction until if fully fit our needs.

It may be generally good to communicate what and how you would like to do something in prior to avoid duplicated effort. But if would be ok if you'd like to present your idea through coding.
